worldedit-core/src/main/java/com/sk89q/worldedit/extension/factory/PatternFactory.java

```diff
@@ -66,6 +70,12 @@ public PatternFactory(WorldEdit worldEdit) {
         register(new PerlinPatternParser(worldEdit));
         register(new RidgedMultiFractalPatternParser(worldEdit));
         register(new BiomePatternParser(worldEdit));
+        register(new Linear2DPatternParser(worldEdit, "#linear2d"));
+        register(new Linear2DPatternParser(worldEdit, "#l2d"));
+        register(new Linear3DPatternParser(worldEdit, "#linear3d"));
+        register(new Linear3DPatternParser(worldEdit, "#l3d"));
```

Comment:
Why are we allowing these to have variable pattern "names"?

Reply:
It's the simplest way to have aliases currently. I can rewrite parts of the RichParser to allow aliases directly if you want tho

Reply:
It's probably best to do it properly, rather than just registering multiple times